examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,535 @@
+      "      {\n",
+      "        \"name\": \"Narita International Airport\",\n",
+      "        \"IATA_code\": \"NRT\"\n",
+      "      },\n",
+      "      {\n",
+      "        \"name\": \"Haneda Airport\",\n",
+      "        \"IATA_code\": \"HND\"\n",
+      "      }\n",
+      "    ]\n",
+      "  },\n",
+      "  \"culture\": {\n",
+      "    \"cuisine\": [\n",
+      "      \"Sushi\",\n",
+      "      \"Ramen\",\n",
+      "      \"Tempura\"\n",
+      "    ],\n",
+      "    \"traditional_events\": [\n",
+      "      \"Cherry Blossom Festival\",\n",
+      "      \"Gion Matsuri\"\n",
+      "    ]\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\", json_mode=True)\n",
+    "@prompt_template(\"Provide information about {city} in JSON format\")\n",
+    "def city_info(city: str): ...\n",
+    "\n",
+    "\n",
+    "response = city_info(\"Tokyo\")\n",
+    "print(response.content)  # This will be a JSON-formatted string"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that not all providers have an explicit JSON mode. For those providers, we attempt to instruct the model to provide JSON; however, there is no guarantee that it will output only JSON (it may start with some text like \"Here is the JSON: ...\"). This is where Output Parsers can be useful.\n",
+    "\n",
+    "For more information on JSON mode, refer to our [Learn documentation on JSON Mode](https://docs.mirascope.io/learn/json-mode)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Response Models\n",
+    "\n",
+    "Response models allow you to structure and validate the output from LLMs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:25.412924Z",
+     "start_time": "2024-09-06T14:00:24.661722Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The capital of Japan is Tokyo\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pydantic import BaseModel\n",
+    "\n",
+    "\n",
+    "class Capital(BaseModel):\n",
+    "    city: str\n",
+    "    country: str\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\", response_model=Capital)\n",
+    "@prompt_template(\"Provide the capital of {country}\")\n",
+    "def get_capital_info(country: str): ...\n",
+    "\n",
+    "\n",
+    "result = get_capital_info(\"Japan\")\n",
+    "print(f\"The capital of {result.country} is {result.city}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more details on response models, including advanced validation techniques, check our [Learn documentation on Response Models](https://docs.mirascope.io/learn/response-models)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Output Parsers\n",
```

Comment:
i think we should update this section so that it is a better follow-up section to JSON mode.

For example:

```python
import json
from mirascope.core import openai, prompt_template


def only_json(response: anthropic.AnthropicCallResponse) -> str:
    json_start = response.content.index("{")
    json_end = response.content.rfind("}")
    return response.content[json_start : json_end + 1]


@openai.call("gpt-4o-mini", json_mode=True, output_parser=only_json)
@prompt_template("Extract {fields} from the following text: {text}")
def json_extraction(text: str, fields: list[str]): ...


json_response = json_extraction(
    text="The capital of France is Paris",
    fields=["capital", "country"],
)
print(json.loads(json_response))
# > {'capital': 'Paris', 'country': 'France'}
```

We can then note that this is particularly important for providers like Anthropic that do not have a strict JSON mode.